chainntnfs/txnotifier.go

```diff
@@ -1749,11 +1749,11 @@ func (n *TxNotifier) dispatchSpendReorg(ntfn *SpendNtfn) error {
 // closes the event channels of all registered notifications that have not been
 // dispatched yet.
 func (n *TxNotifier) TearDown() {
+	close(n.quit)
+
```

Comment:
Separate commit with explanation in commit msg? Looks unrelated to other changes.

Reply:
It already is.

Reply:
Sorry, was viewing the all changes diff